frontend/src/App.vue

```diff
@@ -107,4 +114,9 @@ button {
     padding: 2rem;
   }
 }
+
+.search {
+  position: absolute;
+  z-index: 1000;
```

Comment:
Is this really necessary ?

Reply:
Yes to display the user above the rest of the content